R/npm-scoring.R

```diff
@@ -53,6 +54,27 @@ scoring_function <- function(value, thresholds) {
     return(score)
 }
 
+#' Fruit, Vegetable and nut scorer
+#'
+#' The scoring logic for the percentage of fruit, vegetables and nuts
+#' in a product is different to other scorers. Therefore, we can't use
+#' the generic scoring function and have to write a specific one here.
+#' 
+#' @param value a numeric value of the percentage of fruit, nuts and vegetables
+#' @return a numeric score value
+fruit_veg_nut_scorer <- function(value) {
```

Comment:
This way fruit, veg and nuts percentage is scored is [different ](https://github.com/Leeds-CDRC/NPM-Calculator/blob/39c05814cd9bd7277838a0326e93774b1532fcfd/server.R#L363-L365)to other scoring logics as it can either be 0, 1, 2, or 5. This non linear sequences isn't something we can plug into the `scoring_function` so we write our own scorer here.